sessions-core/src/main/java/com/adyen/checkout/sessions/core/internal/ui/model/SessionParamsFactory.kt

```diff
@@ -34,16 +35,19 @@ object SessionParamsFactory {
 
     private fun SessionDetails.mapToParams(): SessionParams {
         return SessionParams(
-            enableStoreDetails = sessionSetupConfiguration?.enableStoreDetails,
-            installmentConfiguration = SessionInstallmentConfiguration(
-                installmentOptions = sessionSetupConfiguration?.installmentOptions?.map {
-                    it.key to SessionInstallmentOptionsParams(
-                        plans = it.value?.plans,
-                        preselectedValue = it.value?.preselectedValue,
-                        values = it.value?.values,
-                    )
-                }?.toMap(),
-                showInstallmentAmount = sessionSetupConfiguration?.showInstallmentAmount,
+            configuration = SessionConfiguration(
```

Comment:
The field is marked as nullable but it doesn't look like it can be null

Reply:
The way we have it implemented it is not null, but I kept it null in the `SessionConfiguration` to keep it consistent with other fields in `SessionParams` and have them all nullable. Which also allows us to pass null in tests, where we do not need to create an `SessionConfiguration` object and pass null for all of it's arguments.

Though I do not mind making it non-nullable. Let me know what you think.